src/controllers/selectedAccount/selectedAccount.ts

```diff
@@ -7,6 +7,8 @@ import { NetworkId } from '../../interfaces/network'
 import { SelectedAccountPortfolio } from '../../interfaces/selectedAccount'
 import { Storage } from '../../interfaces/storage'
 import { isSmartAccount } from '../../libs/account/account'
+// eslint-disable-next-line import/no-cycle
```

Comment:
I'm a bit scared of import cycles (based on prev experience), is there a way to workaround this or is too complex?